### DISCUSSION TITLE
How to keep storybook from running in the latest upgraded version (6.3.8)

### DISCUSSION THREAD

**Discussion by prodeniser** [storybookjs/storybook#16131] [Help] [answered]
I recently upgraded to storybook 6.3.8 and I can no longer find a way to get storybook to display. I just receive a blank page.

When I manually setback to my previous version:
```
    "@storybook/addon-actions": "^6.2.7",
    "@storybook/addon-essentials": "^6.2.7",
    "@storybook/addon-links": "^6.2.7",
    "@storybook/node-logger": "^6.2.7",
    "@storybook/preset-create-react-app": "^3.1.7",
    "@storybook/react": "^6.2.7",
```

Running storybook still results in storybook being run on 6.3.8, which gives me a blank page. Removing package.lock as well as deleting node_modules still does not allow me to run in the specified version. How can I downgrade my application properly to the version that worked with my application?

**Comment by shilman** [comment, marked as answer]
Here's how to fix a specific version:

```
    "@storybook/addon-actions": "6.2.7",
    "@storybook/addon-essentials": "6.2.7",
    "@storybook/addon-links": "6.2.7",
    "@storybook/node-logger": "6.2.7",
    "@storybook/preset-create-react-app": "^3.1.7",
    "@storybook/react": "6.2.7",
```

**Reply by prodeniser** [reply]
I definitely knew this would do it... I guess in my wave of npm issues etc I couldn't figure it out! thank you!
